### DISCUSSION TITLE
LAI Computation

### DISCUSSION THREAD

**Discussion by sulmonieri** [davidfrantz/force#100] [Q&A]
Does anyone have experience in computing LAI with Level2 ARD or higher level products from Force? I only found publications using SNAP and the Sentinel Toolboxes from ESA. Does someone know if it's possible to feed Force output to SNAP and if yes does anyone have experience with this?

https://step.esa.int/docs/extra/ATBD_S2ToolBox_L2B_V1.1.pdf

**Comment by davidfrantz** [comment]
Hi @sulmonieri ,
I am no expert on LAI, and I am no user of SNAP, but I would assume that using FORCE L2 data with SNAP should work... Not entirely sure what needs to be done to accomplish this though.
David
